internal/wasi/fs.go

```diff
@@ -65,33 +66,61 @@ func (m *MemFS) OpenWASI(dirFlags uint32, path string, oFlags uint32, fsRights,
 		}
 	}
 
-	ret := &memFile{buf: bytes.NewBuffer(buf)}
+	ret := &memFile{buf: buf}
 
 	if fsRights&wasi.R_FD_WRITE != 0 {
 		ret.flush = func(bts []byte) {
-			m.Files[path] = bts
+			m.Files[path] = append([]byte{}, bts...)
```

Comment:
pr comment on what's this change about?

Reply:
thanks, never mind it's just a garbage and I've deleted it. thanks!